stdlib/public/Glibc/CMakeLists.txt

```diff
@@ -1,46 +1,59 @@
 set(sources
-    Glibc.swift
-    Misc.c
+  Glibc.swift
+  Misc.c
 )
 
-set(output_dir "${SWIFTLIB_DIR}/glibc")
+# When cross-compiling the stdlib on Unix platforms, we'll need a separate
+# glibc for each target.
+foreach(SDK ${SWIFT_SDKS})
```

Comment:
To actually make this work, you need to change this condition in the parent `CMakeLists.txt`:

```CMake
if(CMAKE_SYSTEM_NAME STREQUAL "Linux" OR CMAKE_SYSTEM_NAME STREQUAL "FreeBSD")
  add_subdirectory(Glibc)
endif()
```